Multisig/UI/Transaction/Execution/SuccessViewController.swift

```diff
@@ -52,6 +52,10 @@ class TransactionSuccessViewController: UIViewController {
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+        guard let trackingEvent = trackingEvent else {
```

Comment:
why we need this ?

Reply:
in several places the success tracking event is different. I don't have a tracking event yet, so I put this to nil. Otherwise it would crash right?